Gemfile

```diff
@@ -0,0 +1,10 @@
+source "http://rubygems.org"
+
+group :development do
+  gem 'therubyracer'
+  gem 'jekyll'
+  gem 'kramdown'
+  gem 'pygments.rb'
+  gem 'rake'
+  gem 'html-proofer'
+end
```

Comment:
I guess that method could be updated -- if you see where its called from, it's always from within the build directory that the bundle install just above it is called from. My goal was to allow master to be the, well, master of the Gemfile.